<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HriGestureWidget</class>
 <widget class="QWidget" name="HriGestureWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>581</width>
    <height>1124</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="margin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="horizontalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOff</enum>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>563</width>
        <height>1442</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_24">
       <item>
        <widget class="QWidget" name="widget_11" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="margin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Initialize</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBoxInitGestureModule">
            <property name="text">
             <string>Init Gesture Module</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBoxInitHriIOC">
            <property name="text">
             <string>Init HRI IOC</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBoxRecordMotion">
         <property name="title">
          <string>Record Motion</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButtonLoadRecordedMotion">
            <property name="text">
             <string>Load Recorded Motion</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QPushButton" name="pushButtonShowRecordedMotion">
            <property name="text">
             <string>Show Recorded Motion</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QSpinBox" name="spinBoxShowConfigurationRecordedMotion">
            <property name="minimum">
             <number>-1</number>
            </property>
            <property name="maximum">
             <number>1000000000</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButtonSetBeginRM">
            <property name="text">
             <string>Set Start</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButtonSetEndRM">
            <property name="text">
             <string>Set End</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="pushButtonExtarctAndSaveRM">
            <property name="text">
             <string>Extract and save</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonLoadFolder">
            <property name="text">
             <string>Load Folder</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButtonSaveToCSV">
            <property name="text">
             <string>Save To CSV</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButtonLoadFromCSV">
            <property name="text">
             <string>Load From CSV</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="pushButtonLoadTwoFolders">
            <property name="text">
             <string>Load two folders</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="checkBoxPlayNext">
            <property name="text">
             <string>Play Next</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QCheckBox" name="checkBoxRepeat">
            <property name="text">
             <string>Repeat</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Workspace Occupancy</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButtonClassifyMotion">
            <property name="text">
             <string>Classify Motion</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="spinBoxClassToDraw"/>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButtonSetMotionsAndComputeOccupancy">
            <property name="text">
             <string>Set Motions and Occupancy</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Class To Draw</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Simulation</string>
         </property>
         <layout class="QFormLayout" name="formLayout_2">
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBoxDrawOneClassOnly">
            <property name="text">
             <string>Draw one class only</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QCheckBox" name="checkBoxDrawRobotSampledPoints">
            <property name="text">
             <string>Draw robot sampled points</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButtonStartGestureSimulation">
            <property name="text">
             <string>Start</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBoxDrawVoxelOccupancy">
            <property name="text">
             <string>Draw voxel occupancy</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QCheckBox" name="checkBoxDrawHumanSampledPoints">
            <property name="text">
             <string>Draw human sampled points</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="checkBoxDrawHumanTrajectory">
            <property name="text">
             <string>Draw Human Trajectory</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QCheckBox" name="checkBoxDrawCurrentOccupancy">
            <property name="text">
             <string>Draw current occupancy</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Parllel stomp</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_7">
          <item row="0" column="1">
           <widget class="QCheckBox" name="checkBoxMultipleStomps">
            <property name="text">
             <string>Multiple Stomps</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBoxParallelize">
            <property name="text">
             <string>Use Parllelization</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Legible Cost</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QCheckBox" name="checkBoxUseLegibleCost">
            <property name="text">
             <string>Use Legible Cost</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Ioc</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QWidget" name="widget_4" native="true">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="2" column="0">
              <widget class="QCheckBox" name="checkBoxSingleIteration">
               <property name="text">
                <string>Single iteration</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QCheckBox" name="checkBoxLoadSampleFromFile">
               <property name="text">
                <string>Load samples from file</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QCheckBox" name="checkBoxIocExitAfterRun">
               <property name="text">
                <string>Exit after run</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QCheckBox" name="checkBoxUseSimulationDemos">
               <property name="text">
                <string>Use simulation demos</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QCheckBox" name="checkBoxDrawSamples">
               <property name="text">
                <string>Draw samples</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QCheckBox" name="checkBoxGestureDebug">
               <property name="text">
                <string>Debug</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="checkBoxInitSphereCost">
               <property name="text">
                <string>Init Spheres IOC</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QCheckBox" name="checkBoxUserSetPelvisBounds">
               <property name="text">
                <string>User set pelvis bounds</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QCheckBox" name="checkBoxSampleAroundDemo">
               <property name="text">
                <string>Sample around demo</string>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QCheckBox" name="checkBoxNoReplanning">
               <property name="text">
                <string>No Replanning</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QCheckBox" name="checkBoxDrawDemonstrations">
               <property name="text">
                <string>Draw demonstrations</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="checkBoxInitHumanTrajCost">
               <property name="text">
                <string>Init Human Trajectory Cost</string>
               </property>
              </widget>
             </item>
             <item row="6" column="0">
              <widget class="QCheckBox" name="checkBoxUseBaseline">
               <property name="text">
                <string>Use baseline</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="3">
              <widget class="QPushButton" name="pushButtonRunIoc">
               <property name="text">
                <string>Run</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>IOC phase :</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QComboBox" name="comboBoxIoc">
               <item>
                <property name="text">
                 <string>Generate</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Sample</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Compare</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Planning per Feature</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Default</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Monte Carlo</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Simulation</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Save problem</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QComboBox" name="comboBoxPlannerType">
               <item>
                <property name="text">
                 <string>Stomp</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>AStar</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>RRT</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QComboBox" name="comboBoxIK">
               <item>
                <property name="text">
                 <string>Only Motion</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Only IK</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>IK+Motion</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="1" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Sample Iteration</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QSpinBox" name="spinBoxNbOfWayPoints">
               <property name="minimum">
                <number>4</number>
               </property>
               <property name="maximum">
                <number>300</number>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Stddev Motion</string>
               </property>
              </widget>
             </item>
             <item row="3" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxStandardDev">
               <property name="decimals">
                <number>7</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="spinBoxSampleIteration">
               <property name="maximum">
                <number>1000</number>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Nb of way points</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Spheres Power</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QSpinBox" name="spinBoxSphereLayerToDraw"/>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxSpherePower"/>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Layer To Draw</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Stddev IK</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QDoubleSpinBox" name="doubleSpinBoxStandardDevIK"/>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <layout class="QFormLayout" name="formLayout_3">
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButtonDetours">
            <property name="text">
             <string>Generatge Detours</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Classification</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="1" column="2">
           <widget class="QPushButton" name="pushButtonClassifyBothBad">
            <property name="text">
             <string>Both Bad</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QPushButton" name="pushButtonClassifyBad">
            <property name="text">
             <string>Bad</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButtonClassifyGood">
            <property name="text">
             <string>Good</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QPushButton" name="pushButtonClassifyInit">
            <property name="text">
             <string>Init</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
